css/css-anchor-position/anchor-position-dynamic-002.html

```diff
@@ -41,14 +41,14 @@
     <div class="target" style="width: anchor-size(--a2 width)" data-expected-width=5></div>
   </div>
 <script>
-checkLayout('.target');
+checkLayout('.target', false);
 container.classList.add('after');
 for (const element of document.getElementsByClassName('target')) {
   if (element.dataset.offsetX === '5')
     element.dataset.offsetX = '10';
   if (element.dataset.expectedWidth === '5')
     element.dataset.expectedWidth = '10';
 }
-checkLayout('.after .target');
+checkLayout('.after .target', true);
```

Comment:
This follows the same pattern as other tests (e.g. `css-grid`) where `checkLayout()` is called multiple times, with changes in between.